### ISSUE TITLE
RC Testing for Balloons and Static Electricity PhET-iO version 1.2.0-phetiorc.1

### ISSUE THREAD

**Issue by jbphet** [phetsims/tasks#770]
Link: http://www.colorado.edu/physics/phet/dev/html/balloons-and-static-electricity/1.2.0-phetiorc.1/wrappers/index/

Test Matrix: https://docs.google.com/spreadsheets/d/1iUMWBd4HMZGbHMM7An4kh_YcWo2dqAodJUI-1dOoMuc/edit#gid=2

This version is going to be shared with various collaborators, so should be tested pretty thoroughly.

Also, the process of instrumenting the simulation revealed some bugs that have been fixed in this version, but the change altered the behavior of the simulation versus the currently published version.  I have discussed these issues with @phet-steele so he is already pretty aware of them.  We should make sure that the new behavior is acceptable.

Assigning to @ariel-phet for priority and scheduling.  @emily-phet requested that the sim be ready to share by the end of this week, which would be Friday Jan 27 2017, if possible.

**Comment by ariel-phet** [comment]
This task is also a "top" priority along with #768 

@phet-steele it looks like #768 is mainly verification so hopefully the testing for BASE can also proceed soon.

**Comment by phet-steele** [comment]
@jbphet done.

**Comment by jbphet** [comment]
Thanks.  I'll start tackling the issues that were found.  Closing.
